### DISCUSSION TITLE
export and import javascripts ignored errors

### DISCUSSION THREAD

**Discussion by adamsvibeREAL** [Dynatrace/dynatrace-configuration-as-code#778] [Ideas]
Hello,

We are using Monaco to backup our configurations.

We also have multiple instance of the same application ( appDev1 , appDev2 , appdDev3 , appProd by example ) and we would like to make sure that every instances have the same configurations .

Thanks to Monaco, we can export / import Application configuration.

Except for one important "thing" for us : javascript errors ignored

Do you plan to add this to monaco , please ?

I have posted this idea here : 
https://community.dynatrace.com/t5/Dynatrace-product-ideas/Monaco-export-and-import-javascripts-ignored-errors/idc-p/198768#M33901

Thank you.

Regards,

Adama

**Comment by UnseenWizzard** [comment]
Hi @adamsvibeREAL, actually the community is becoming our preferred way for product ideas for monaco - I've replied there :) 

-- 
for completeness: 


If I understand your request correctly, "Javascript errors ignored" is part of the configuration for a specific Application?

I think in terms of API that is part of /applications/web/{id}.


Generally, we are planning to support configurations scoped to specific entities - like this one for applications - in the future.
